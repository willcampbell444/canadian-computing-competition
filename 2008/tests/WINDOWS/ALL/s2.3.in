99
206
777
0
