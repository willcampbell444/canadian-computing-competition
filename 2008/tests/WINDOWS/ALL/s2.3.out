30757
133301
1896633
